### PR TITLE
check changes

### PR DESCRIPTION
**Pull request by Bhargavijanga58**

### CODE REVIEW

File: docs/host.md [hunk 1]
```diff
@@ -5,63 +5,68 @@
...
+ 
 
 ## Create a Host
 
-1. Click the plus button after `Hosts` on the right panel.
+1. Click the `New Host` button at the `Hosts` on the left panel.
+ 
+2. Leave the `Host Type` as default, add `Host Name`, `Host IP`, `Port`, and `User Name` of your server in the shown dialog box.
 
-2. Input `Host Name`, `Host IP`, `Port`, and `User Name` of your server in the shown dialog. Here is an example:
```
Comment (gokulprathin8) [inline docs/host.md]: Is the button name changed? 
from Host Type to Host Name in UI ?

Comment (Bhargavijanga58) [reply docs/host.md]: No there is another button as Host type with the different hosts, and the rest of the buttons are same.

**Review by gokulprathin8 — APPROVED** [review]
Good Job!
LGTM
Open the same PR @ ZihengSun's Repo

File: docs/host.md (hunk 1 above)
Comment (gokulprathin8) [reply docs/host.md]: 👍🏻

**Pull request closed without merging**